hooks/pre/30-upgrade.rb

```diff
@@ -80,6 +80,22 @@ def fix_katello_settings_file
   end
 end
 
+def mongo_mmapv1_check
+  custom_hiera = '/etc/foreman-installer/custom-hiera.yaml'
```

Comment:
Hardcoding this path is generally not a good idea, but I guess it already happens a lot.

Reply:
:crying_cat_face: